tests/hotplug/memory.go

```diff
@@ -257,5 +257,63 @@ var _ = Describe("[sig-compute][Serial]Memory Hotplug", decorators.SigCompute, d
 			Expect(vmi.Spec.Domain.Resources.Requests.Memory().Value()).To(Equal(newGuestMemory.Value()))
 		})
 
+		It("should successfully hotplug memory when adding guest.memory to a VM", func() {
```

Comment:
Don't you always set guest now? I am not sure if you actually test wat you want

Reply:
I always set in the VMI, not in the VM. This tests that you can add it to a VM which didn't have it and will still trigger hotplug.